ocaml/xapi/import.ml

```diff
@@ -234,6 +234,27 @@ let assert_can_live_import __context rpc session_id vm_record =
   if vm_record.API.vM_power_state = `Running || vm_record.API.vM_power_state = `Paused
   then assert_memory_available ()
 
+(* Assert that the local host, which is the host we are live-migrating the VM to,
+ * has free capacity on a PGPU from the given VGPU's GPU group. *)
+let assert_can_live_import_vgpu ~__context vgpu_record =
+  let host = Helpers.get_localhost ~__context in
+  let local_pgpus = Db.PGPU.get_refs_where ~__context ~expr:Db_filter_types.(And
+    (Eq (Field "GPU_group", Literal (Ref.string_of vgpu_record.API.vGPU_GPU_group)),
```

Comment:
No, but `vgpu_record` represents the VGPU record that we will create on the receiving pool, if all goes well. The reference fields in there have already been resolved for the receiving pool by the import handlers of the relevant objects.